### PR TITLE
display a message when events are added to users favs list

### LINKED ISSUE #99: Display a message 'The event is added to your favs list' once user added the event

### PR DESCRIPTION
**Pull request by SachiGoto**
### Feature Description

It shows 'The event is added to your favs list' once uses clicked the favourite icon. 
Screenshot 2023-11-07 at 8 59 13 PM
Screenshot 2023-11-07 at 8 59 26 PM
Screenshot 2023-11-07 at 8 59 46 PM
Screenshot 2023-11-07 at 8 59 59 PM

### Related Issue
close #99 

### Checklist before requesting a review

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/app/(events)/map/components/GoogleMaps/index.tsx [hunk 1]
```diff
@@ -203,6 +212,21 @@ export const GoogleMaps = ({ setMapLoaded }: GoogleMapsProps) => {
       <div className={classes.container}>
         <Calendar date={date} setDate={setDate} />
       </div>
+      {eventAdded && (
+        <div className={classes.overlay}>
+          <Notification
+            style={{ padding: "3rem" }}
+            className={`${classes.notification}`}
+            onClick={handleIconClick}
+            icon={checkIcon}
```
Comment (keento0809) [inline src/app/(events)/map/components/GoogleMaps/index.tsx:218]: @SachiGoto This way `onClick={handleIconClick}` is absolutely ok, but we can also do like this to reduce the code lines. And then we can delete `handleIconClick` function

```typescript
...
onClick={() => setEventAdded(false)}
```

**Review by keento0809 — COMMENTED** [review]
@SachiGoto Thank you for adding the notification messages! I was also planning the same thing, Lol So I can add another notification message when a favEvent is deleted.

I left a comment on your PR, so plz take a look at it when you have time!

File: src/app/(events)/map/components/GoogleMaps/index.tsx (hunk 1 above)
Comment (SachiGoto) [reply src/app/(events)/map/components/GoogleMaps/index.tsx:218]: Ah that's a really good point! I tend to make things unnecessary complicated haha 
Thank you so much for the suggestion!

Comment (SachiGoto) [reply src/app/(events)/map/components/GoogleMaps/index.tsx:218]: just updated my code!

**Review by Lada496 — APPROVED** [review]

**Review by keento0809 — APPROVED** [review]
@SachiGoto Thanks for the fixment! LGTM! 👍

**Pull request merged**